docs/user/convert/jdbc.rst

```diff
@@ -1,28 +1,52 @@
 .. _jdbc_converter:
 
-Parsing SQL Data
-----------------
+JDBC Converter
+==============
 
-GeoMesa includes a converter designed to read SQL data using standard JDBC libraries. The connection string
-is specified using the ``connection`` element.
+The JDBC converter allows you to create SimpleFeatures directly from a SQL select statement against an
+eixsting database, using standard JDBC libraries. To use the JDBC converter, specify ``type = "jdbc"``
+in your converter definition.
 
+Configuration
+-------------
+
+The JDBC connection string used to connect to the database must be specified using the ``connection`` element.
```

Comment:
Can we give a more complete example here?

Reply:
added